src/components/TemplateModal.jsx

```diff
@@ -0,0 +1,74 @@
+"use client";
+import { Modal, Button, Flex, Divider } from "@mantine/core";
+import { useState } from "react";
+import colors from "@/style/colors";
+import clsx from "clsx";
+import { CREAT_REPORT_OPTIONS } from "@/constants";
+
+export const TemplateModal = ({ opened, close }) => {
+  const [selectedType, setSelectedType] = useState();
+
+  return (
+    <Modal.Root opened={opened} onClose={close} size="lg" centered>
+      <Modal.Overlay />
+      <Modal.Content>
+        <div className="m-[20px]">
```

Comment:
```suggestion
        <div className="m-5">
```
อันไหนใช่ `rem` ได้ก็ใช้ของมันดีกว่า แต่อันนี้ไม่ต้องแก้นะๆ